src/Entity/Reader.php

```diff
@@ -1,27 +1,34 @@
 <?php
+
+namespace BookReviews\Entity;
+
 /**
- * Created by PhpStorm.
- * User: ioana
- * Date: 11/27/2018
- * Time: 4:10 PM
+ * Class Reader
+ * @package BookReviews\Entity
  */
-
 class Reader
 {
-    protected  $id;
```

Comment:
Properties need comments in the format
```
/**
* @var {type}
*/
```
The variable name is not required when putting the comment outside of a method.